onnxruntime/hosting/http/http_server.cc

```diff
@@ -29,42 +25,44 @@ namespace hosting {
 using handler_fn = std::function<void(std::string, std::string, std::string, HttpContext&)>;
 
 App::App() {
-  // TODO: defaults should come from central place
-  address_ = boost::asio::ip::make_address_v4("0.0.0.0");
-  port_ = 8080;
-  threads_ = std::thread::hardware_concurrency();
+  http_details.address = boost::asio::ip::make_address_v4("0.0.0.0");
+  http_details.port = 8080;
```

Comment:
Better use the same default as we used in the configuration.

Reply:
Changed to 8001.